tests/session/cmd/test_single.py

```diff
@@ -0,0 +1,20 @@
+import sys
```

Comment:
I believe this belongs into https://github.com/tox-dev/tox/blob/rewrite/tests/session/cmd/test_sequential.py

Reply:
Thanks - moved it with the next commit/push.